hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -1367,8 +1460,9 @@ public TransactionBody.Builder body(final Bytes input, final UnaryOperator<byte[
 
 			final var nestedInput = input.slice(24);
 			super.transferOp = switch (nestedInput.getInt(0)) {
-				case ABI_ID_ERC_TRANSFER -> decoder.decodeErcTransfer(nestedInput, tokenID, callerAccountID,
-						aliasResolver);
+				case ABI_ID_ERC_TRANSFER -> decoder.decodeErcTransfer(nestedInput, tokenId, callerAccountID, aliasResolver);
```

Comment:
We can rename `decodeErcTransfer` -> `decodeERCTransfer` to be compliant with `decodeERCTransferFrom()`.